.github/workflows/node.js.yml

```diff
@@ -1,7 +1,6 @@
 # This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
 # For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
-
-name: Node.js CI
+name: Checkout, install deps & test
```

Comment:
Since we are using the squash merging strategy in this project, could you split this out into a separate PR so that each one could focus on doing one thing?

Reply:
Done, can mail that one out later.